services/apps/portal/src/application/application-document/application-document.controller.ts

```diff
@@ -43,6 +46,9 @@ export class ApplicationDocumentController {
     @Param('documentType') documentType: string,
     @Req() req,
   ): Promise<ApplicationDocumentDto> {
+    //Verify Access
+    await this.applicationService.getIfCreator(fileNumber, req.user.entity);
```

Comment:
You could create a wrapper method 'verifyAccess' for 'getIfCreator' that does not return any value and simple calls the 'getIfCreator'. This way you do not need to add a comment on every call

Reply:
Good idea, done!